src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const Calculator = () => {
+  const value = 0;
+  return (
+    <>
+      <div className="calculator">
+        <section className="monitor">
+          <p>{value}</p>
+        </section>
+        <section className="calcbtnContainer">
+          <button type="button">AC</button>
+          <button type="button" className="twosigne">-/+</button>
+          <button type="button" className="pourcent">%</button>
+          <button type="button" className="orange">&#247;</button>
+          <button type="button">7</button>
+          <button type="button">8</button>
+          <button type="button">9</button>
+          <button type="button" className="orange">&#215;</button>
+          <button type="button">4</button>
+          <button type="button">5</button>
+          <button type="button">6</button>
```

Comment:
- [ ] According to the requirements Calculator.js should have at least 1 child component